tools/stress-cluster/services/Stress.Watcher/ci.yml

```diff
@@ -25,4 +25,5 @@ extends:
   template: /eng/pipelines/templates/stages/archetype-sdk-tool-dotnet.yml
   parameters:
     ToolDirectory: tools/stress-cluster/services/Stress.Watcher
+    DotNetCoreVersion: "7.0.x"
```

Comment:
This is no longer necessary due to:
- https://github.com/Azure/azure-sdk-tools/pull/5379